### ISSUE TITLE
EnvSet update causes Install Error during migrations

### ISSUE THREAD

**Issue by kiltedup** [pyrocms/pyrocms#5047]
Following error occurs during fresh install.

```
33/61 Seeding: Blocks Module
In Container.php line 960:
  Target [Anomaly\SettingsModule\Setting\Contract\SettingRepositoryInterface] is not instantiable.
```

Issue appears to be introduced by commit :

https://github.com/anomalylabs/streams-platform/commit/228a0a2ef7321d7b0748d7bc0590e06eec864a65

Pull that commit and the installer completes as expected.

**Comment by RyanThompson** [comment]
What is the content of the .env file at this point?

**Comment by kiltedup** [comment]
sorry - didn't check the contents of the .env file but it was as per the installation docs prior to running install --ready

**Comment by RyanThompson** [comment]
Can you share what values you had in your .env file? 

I ran `composer create-project pyrocms/pyrocms` and then `php artisan install` and let the system handle it, and it worked. I suspect maybe something in the `.env` wasn't agreeable.

**Comment by kiltedup** [comment]
OK - tried again with `php artisan install` and it worked.

What I had been doing (and always) was using `php artisan install --ready` with the .env from the documentation ( [https://pyrocms.com/documentation/pyrocms/3.7/getting-started/installation](url) ) - edited of course. This was fine till recently as I had done a couple if installs recently this way.

Looks like the issue was `INSTALLED=false` not included in that .env from the docs. I added it, and tried again and all was fine. 

So probably no issue and maybe just that documentation needs the installed libne from the .env adding.

**Comment by edster** [comment]
Fixed: e0133c47dcc5b179e32a1eb755d5782563cf3fea, thanks for that.
